sdk/react/panels/ItemSelectionList.js

```diff
@@ -21,11 +21,12 @@ class ItemSelectionList extends React.Component {
   };
 
   renderItems = (items) => {
-    const renderedItems = items.map((item, index) => (
-        this.props.itemRender(item, index)
-    ));
-
-    return (renderedItems);
+    if (typeof(items) !== "undefined") {
+      const renderedItems = items.map((item, index) => (
+          this.props.itemRender(item, index)
+      ));
+      return (renderedItems);
+    }
```

Comment:
doesn't this one need another return value in case it's `undefined`?

Reply:
@rcvrgs Not sure what we can return for undefined value. Currently, it shows blank screen. 
Output of renderItems() is passed to ScrollView which is expected not show anything.

Tried to "return items" and gave me same result.

Reply:
@rcvrgs @pilievOoyala  Updated now.